Cargo.toml

```diff
@@ -44,6 +44,7 @@ num-derive = "0.4.0"
 num-traits = "0.2.6"
 p256 = { version = "^0.13", features = ["ecdsa"] }
 p384 = { version = "^0.13", features = ["ecdsa"] }
+libsecp256k1 = "0.7.1"
```

Comment:
`k256` provides better performance than `libsecp256k1` and is sourced from @RustCrypto like `p256` and `p384`

Reply:
Didn't noticed in the first place. Just replaced it with `k256`.